argo-cd-apps/overlays/development/delete-applications.yaml

```diff
@@ -121,12 +121,6 @@ $patch: delete
 ---
 apiVersion: argoproj.io/v1alpha1
 kind: ApplicationSet
-metadata:
-  name: kyverno
-$patch: delete
----
-apiVersion: argoproj.io/v1alpha1
-kind: ApplicationSet
```

Comment:
this will deploy to development with 3 replicas for each controller. That may be too much. I'd suggest to add an overlay for development where we explicitly set 1 replica for each Kyverno controller. We could also disable Report ones.

Reply:
Fixed in 4b1460fe.